week-01-introduction.tex

```diff
@@ -4,6 +4,7 @@
 \usepackage{listings}
 \usepackage{tikz}
 \usepackage{color}
+\usepackage{microtype}
```

Comment:
I added `microtype` because it helps clean up over/underfull hboxes. Also it's just good style IMHO—why even use computers if we don't waste the energy on clawing back some of that last 1% of typographical pleasantness?

When I compiled this, I used `xelatex` and it worked fine. Not sure if you were using a different tool.